### ISSUE TITLE
Server url doesn't support variable parsing.

### ISSUE THREAD

**Issue by yorkish** [42Crunch/vscode-openapi#214]
the OpenAPI specs states that the servers definitions supports variables templating but this VSCode extension doesn't honour this feature.

https://swagger.io/docs/specification/api-host-and-base-path/

![open-api-vscode](https://user-images.githubusercontent.com/12158934/231811228-c1b5b1a5-05ad-43b4-845d-cdbfb70171b0.png)

**Comment by ak1394** [comment]
Correct, this is something for us to fix. We'll try to address it in the next version.

**Comment by anthony-42crunch** [comment]
@ak1394 
The variable does appear to be correctly substituted with the default value at execution time, 
but the variable name rather than the value is displayed in the URL field, 
and there doesn't seem to be way to select a value other than the default (without modifying the OAS)
